src/jshint.js

```diff
@@ -4242,7 +4242,18 @@ var JSHINT = (function() {
       // ImportClause :: ImportedDefaultBinding
       this.name = identifier();
       addlabel(this.name, { type: "unused", token: state.tokens.curr });
-    } else if (state.tokens.next.id === "*") {
+      if (state.tokens.next.value === ",") {
+        // ImportClause :: ImportedDefaultBinding , NameSpaceImport
```

Comment:
@rwalrdron can you explain the ImportDefaultBinding grammar? It doesn't make a whole lot of sense for that to be followed by comma NameSpaceImport (or comma NamedImports) --- and there's no example of this in section 15.2.1.15, which could just be an oversight but the grammar just looks wrong